### PR TITLE
Add minimizer metric

### PR DESCRIPTION
**Pull request by mpherman2**
Add log output to report number of times minimizer runs testcase in order to compare minimizers better in the future.

### CODE REVIEW

File: src/python/bot/minimizer/minimizer.py [hunk 1]
```diff
@@ -216,8 +217,9 @@ def _report_progress(self):
       return
 
     self.last_progress_report_time = time.time()
-    message = '%d/%d tokens remaining.' % (len(self.get_required_tokens()),
-                                           len(self.required_tokens))
+    message = '%d/%d tokens remaining. - %d runs executed so far.' % (len(
```
Comment (inferno-chromium) [inline src/python/bot/minimizer/minimizer.py]: no need of - in "- %d"

Comment (mpherman2) [reply src/python/bot/minimizer/minimizer.py]: Removed the -

**Pull request merged**